### PR TITLE
Replace magic numbers with TILE_WIDTH

### PR DESCRIPTION
**Pull request by Idain**

### CODE REVIEW

File: engine/battle/trainer_huds.asm [hunk 1]
```diff
@@ -182,9 +182,9 @@ LinkBattle_TrainerHuds:
 	ld de, wPartyCount
 	call StageBallTilesData
 	ld hl, wPlaceBallsX
-	ld a, 10 * 8
+	ld a, 10 * TILE_WIDTH
 	ld [hli], a
-	ld [hl], 8 * 8
+	ld [hl], 8 * TILE_WIDTH
 	ld a, 8
```
Comment (Rangi42) [inline engine/battle/trainer_huds.asm:188]:
```suggestion
	ld a, TILE_WIDTH
```

File: engine/battle/trainer_huds.asm [hunk 2]
```diff
@@ -36,9 +36,9 @@ ShowOTTrainerMonsRemaining:
 	call StageBallTilesData
 	; ldpixel wPlaceBallsX, 9, 4
 	ld hl, wPlaceBallsX
-	ld a, 9 * 8
+	ld a, 9 * TILE_WIDTH
 	ld [hli], a
-	ld [hl], 4 * 8
+	ld [hl], 4 * TILE_WIDTH
 	ld a, -8
```
Comment (Rangi42) [inline engine/battle/trainer_huds.asm:42]:
```suggestion
	ld a, -TILE_WIDTH
```

Comment (Idain) [reply engine/battle/trainer_huds.asm:42]: Thanks for the observation!

**Comment by Idain** [comment]
@Rangi42 or @vulcandth, feel free to further review this PR when you have time.

**Pull request merged**